server/dialect-idms/src/main/java/org/eclipse/lsp/cobol/dialects/idms/MessageServiceParser.java

```diff
@@ -21,9 +21,6 @@
 import org.antlr.v4.runtime.Parser;
```

Comment:
Doubt: Should we have a [MessageServiceParser.java](https://github.com/eclipse/che-che4z-lsp-for-cobol/pull/1629/files#diff-9232a81b61df19cd26f68735d97155b29b7915e4cb818e12fc2c3c5d872f98e7) in common module , with diff validation support ? 
Not for this PR , just a doubt